.gitignore

```diff
@@ -3,3 +3,4 @@
 rosserial_server/doc/
 *.out
 *.geany
+
```

Comment:
Try not to make unrelated whitespace changes. It clutters up the diffs and makes reviewing and merging much harder.